### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by darhan1168**
Do homework1 with classes and console app

### CODE REVIEW

File: Homework1/Homework1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            ManagementRestaurant managementRestaurant = new ManagementRestaurant();
+
+            while (true)
+            {
+                Console.WriteLine("Choose what you want (1 - add, 2 - watch, 3 - create order, 4 - stop app)");
+                string? answerUserMethods = Console.ReadLine();
```
Comment (JustDimus) [inline Homework1/Homework1/Program.cs:12]: Why did you add '?' symbol at the end of string class name?

File: Homework1/Homework1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+                        string? answerUserAdd = Console.ReadLine();
+
+                        switch (answerUserAdd)
+                        {
+                            case "1":
+                                Console.WriteLine("Enter name of dish");
+                                string? dishName = Console.ReadLine();
+                                
+                                Console.WriteLine("Enter cost of dish");
+                                int dishCost = Int32.Parse(Console.ReadLine() ?? string.Empty);
```
Comment (JustDimus) [inline Homework1/Homework1/Program.cs:27]: Console.ReadLine returns enpty string if user entered no symbols. Why did you use ?? statement?

**Review by JustDimus — APPROVED** [review]

File: Homework1/Homework1/Program.cs (hunk 1 above)
Comment (darhan1168) [reply Homework1/Homework1/Program.cs:12]: Because without '?' he swears that there may be a null value

File: Homework1/Homework1/Program.cs (hunk 2 above)
Comment (darhan1168) [reply Homework1/Homework1/Program.cs:27]: Reader says fix it like this
